src/haloniumpkg/webdriver.nim

```diff
@@ -1645,7 +1642,7 @@ proc findElements*(element: Element, selector: string, strategy = CssSelector):
       getSelectorParams(element.session, selector, strategy),
       stopOnException = false
     )
-    for elementNode in response["value"].to(seq[JsonNode]):
+    for elementNode in response["value"].items:
```

Comment:
Is this correct?

Reply:
To answer, yes `to(seq[JsonNode])` only accepts `JArray` node and so does `items`